src/app/psbt.ts

```diff
@@ -331,6 +338,167 @@ const Psbt = function (config) {
             return psbtModule.broadcastPsbt(psbt);
         },
 
+        signMultipleUtxosPsbtForSend: async ({
+            address, pubKey, selectedUtxos,
+            ownedUtxos, destinationBtcAddress, sendFeeRate
+        }) => {
+            const utxosWithInscription = selectedUtxos.filter(utxo => utxo.inscriptionId);
+            const utxosWithoutInscription = selectedUtxos.filter(utxo => !utxo.inscriptionId);
+            if (utxosWithInscription.length === 0 && utxosWithoutInscription.length === 0) {
+                throw new Error('At least one ordinal or utxo is required.');
+            }
+            const provider = SessionStorage.get(SessionsStorageKeys.DOMAIN);
+            if (provider !== 'alby') {
+                throw new Error('Signing not supported.');
+            }
+
+            const spendableUtxos = ownedUtxos.filter(utxo => !selectedUtxos.some(ownedUtxo => ownedUtxo.txid === utxo.txid))
```

Comment:
actually for consistency, I think we should use this as the definition of "utxosWithoutInscription"

because the idea is that, for safety purposes, if a UTXO is smaller than 10,000 sats, it will appear as a separate UTXO in the output

Also it'll be good to have these sorted.